quill-sql/src/test/scala/io/getquill/QuotationTest.scala

```diff
@@ -370,7 +370,7 @@ class QuotationTest extends Spec with Inside {
       qqq must matchPattern {
```

Comment:
TODO @masonedmison I still need to convince myself that this is a sound change

Reply:
Looks alright to me. We can talk about it when I’m back in a couple days.